genrules.cpp

```diff
@@ -5049,8 +5049,26 @@ int Game::GenRules(const AString &rules, const AString &css,
 	f.ClassTagText("div", "rule", "");
 	f.LinkRef("forget");
 	f.TagText("h4", "FORGET [skill]");
-	temp = "Forget the given skill. This order is useful for normal units "
-		"who wish to learn a new skill, but already know a different skill.";
+	temp = "Forget the given skill. This order is useful for ";
+	if (Globals->SKILL_LIMIT_NONLEADERS) {
+		temp += "normal units who wish to learn a new skill, but already "
+			"know a different skill and for ";
```

Comment:
we probably can rephrase it somehow / split it into multiple sentences? It seems long hard to read sentence

Reply:
Agree. I have split it into three sentences, which makes it a bit easier to read.